jfrog-cli/jfrog/artifactory_test.go

```diff
@@ -944,6 +944,27 @@ func TestExcludeBrokenSymlink(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+// Testing uploading broken symlinks.
+// Uploaded symlinks are not verified by default, so all symlinks should be uploaded
+func TestUploadBrokenSymlink(t *testing.T) {
```

Comment:
I suggest modifying this test from a "TestUploadBrokenSymlink" to a "TestValidateSymlink".
Basically, the test can upload a broken symlink (the same way it does now) and then, verify that this symlink fails to be downloaded with the *--validate-symlink* option and succeeds to download it without the option.
This way, instead of testing a bug fix, we're testing a real functionality of JFrog CLI.